src/api/app/policies/project_policy.rb

```diff
@@ -35,4 +34,13 @@ def can_create_package_in?
   def local_project_and_allowed_to_create_package_in?
     local? && can_create_package_in?
   end
+
+  def update_or_delete
```

Comment:
The name sounds like this method would delete or update something. I would rename it to something like `write_permissions?`